deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
+if [ -z "$AVAIL_BOX" ]; then
+  vagrant box add chef/centos-7.0 --provider virtualbox
+fi
+
+echo "Generating 'tachyon-dev.box' based on box '$BASE_BOX' ..."
+vagrant box repackage $BASE_BOX virtualbox 1.0
+mv package.box tachyon-dev.box
```

Comment:
@yuzhu @uronce-cc I am cleaning this thing up. Any specific reason for putting maven tarball in to a synced folder  (guest's /vagrant/shared)? This guest's folder get mapped into different host's folders in different "vagrant up" runs.